docs/source/deployment/airflow_astronomer.md

```diff
@@ -2,161 +2,146 @@
+  filepath: data/02_intermediate/X_test.csv
 
-* Create a `conf/airflow` directory in your Kedro project
-* Create a `catalog.yml` file in this directory with the following content
+y_train:
+  type: pandas.CSVDataset
+  filepath: data/02_intermediate/y_train.csv
 
-```yaml
-example_iris_data:
+y_test:
   type: pandas.CSVDataset
-  filepath: data/01_raw/iris.csv
-example_train_x:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_train_x.pkl
-example_train_y:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_train_y.pkl
-example_test_x:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_test_x.pkl
-example_test_y:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_test_y.pkl
-example_model:
-  type: pickle.PickleDataset
-  filepath: data/06_models/example_model.pkl
-example_predictions:
-  type: pickle.PickleDataset
-  filepath: data/07_model_output/example_predictions.pkl
+  filepath: data/02_intermediate/y_test.csv
 ```
 
-This ensures that all datasets are persisted so all Airflow tasks can read them without the need to share memory. In the example here we assume that all Airflow tasks share one disk, but for distributed environment you would need to use non-local filepaths.
+In the example here we assume that all Airflow tasks share one disk, but for distributed environment you would need to use non-local filepaths.
+
+4. Open `conf/logging.yml` and modify the `root: handlers` section to `[console]` at the end of the file. By default, Kedro use the rich library to enhance log output with sophisticated formatting. However, some deployment systems, including Airflow, may encounter difficulties with the rich format. Therefore, we're adjusting the logging to a simpler console version. For more information on logging in Kedro, you can refer to the [Kedro docs](https://docs.kedro.org/en/stable/logging/index.html).
 
-#### Step 2. Package the Kedro pipeline as an Astronomer-compliant Docker image
+```shell
+root:
+  handlers: [console]
+```
 
-* **Step 2.1**: Package the Kedro pipeline as a Python package so you can install it into the container later on:
+5. Package the Kedro pipeline as a Python package so you can install it into the Airflow container later on:
 
 ```shell
 kedro package
 ```
 
 This step should produce a wheel file called `new_kedro_project-0.1-py3-none-any.whl` located at `dist/`.
 
-* **Step 2.2**: Add the `src/` directory to `.dockerignore`, as it's not necessary to bundle the entire code base with the container once we have the packaged wheel file.
+6. Convert the Kedro pipeline into an Airflow DAG with `kedro airflow`
 
 ```shell
-echo "src/" >> .dockerignore
+kedro airflow create --target-dir=dags/ --env=airflow
 ```
 
-* **Step 2.3**: Modify the `Dockerfile` to have the following content:
+This step should produce a .py file called `new_kedro_project_dag.py` located at `dags/`.
+
+### Deployment process with Astro Airflow
```

Comment:
I would again add a bit of an introduction paragraph before diving into the steps to explain what we'll be doing in this part of the tutorial.